packages/loader/container-loader/src/loader.ts

```diff
@@ -332,18 +332,17 @@ export class Loader implements IHostLoader {
 	public async resolve(request: IRequest, pendingLocalState?: string): Promise<IContainer> {
 		const eventName = pendingLocalState === undefined ? "Resolve" : "ResolveWithPendingState";
 		return PerformanceEvent.timedExecAsync(this.mc.logger, { eventName }, async () => {
-			const resolved = await this.resolveCore(
+			return this.resolveCore(
 				request,
 				pendingLocalState !== undefined ? JSON.parse(pendingLocalState) : undefined,
 			);
-			return resolved.container;
 		});
 	}
 
 	private async resolveCore(
 		request: IRequest,
 		pendingLocalState?: IPendingContainerState,
-	): Promise<{ container: Container; parsed: IParsedUrl }> {
```

Comment:
I believe the return of the parsed URL is just leftovers from when the loader used to cache containers (using the resolved url as the cache key).